docs/dataformats/observation_lists.rst

```diff
@@ -22,23 +22,48 @@ This has the advantage that everyone can work with whatever tool they like. Here
 * `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
 * Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
 
-A run list must have at least a column called ``Run``::
+A run list must have at least a column called ``OBS_ID``::
  
-   Run
+   OBS_ID
    42
    43
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is::
+
+   OBS_ID: observation ID as an integer (starting at 0 or 1? does it matter?)
+   RA: pointing position right ascension in equatorial (ICRS) coordinates
+   DEC: pointing position declination in equatorial (ICRS) coordinates
+   AZ: average azimuth angle during the observarion
+   ALT: average altitude angle during the observarion
+   MUON_EFFICIENCY: average muon efficiency of the telescopes
+   TIME_START: start time of the observation stored as number of seconds after the time reference in the header
+   TIME_STOP: end time of the observation in the same forma as TIME_START
+   TIME_OBSERVATION: duration of the observation
+   TIME_LIVE: duration of the observation without dead time
+   TRIGGER_RATE: average trigger rate of the system
```

Comment:
This optional / required question is related to the question whether we have one or two classes representing observation tables / data store index tables. I still don't know what is best. If there's only one class, I agree that only `OBS_ID` should be required.

My suggestion would be you continue implementing functionality (obs selection and grouping) for a few days and then maybe it will become clearer what is best?